metacat-common/src/main/java/com/netflix/metacat/common/json/MetacatJsonException.java

```diff
@@ -17,10 +17,28 @@
  */
 package com.netflix.metacat.common.json;
 
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * Metacat JSON utility related exception.
  */
+@Getter
+@Setter
 public class MetacatJsonException extends RuntimeException {
```

Comment:
There's really no reason for an exception class to have a setter. It's immutable after being thrown. Make the input JSON final.